<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagingSpec</class>
 <widget class="QDialog" name="ManagingSpec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_userid">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>user id :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_inputID">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>inputID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>226</width>
     <height>232</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>University</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Major</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>학점</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>TOEIC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>OPIC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>TOS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>자격증 개수</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>연수</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>인턴 경험</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_university"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_major"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_grade"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_toeic"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_opic"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_tos"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_numberofcirtificate"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_langstudy"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_numberofintern"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼둥근헤드라인</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>나의 스펙 관리</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
